data/korath/korath missions.txt

```diff
@@ -257,3 +257,223 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+
+substitutions
+	<EDC-ship> "Esketari"
+
+mission "Efreti: Distress Call 1"
+	name "Investigate distress call"
+	description "The Quarg have asked you investigate an Efreti distress call in <destination>."
+	source "Karek Fornati"
+	stopover "Seleptra Nak"
+	destination "Laki Nemparu"
+	passengers 40
+	landing
+	minor
+	random < 30
+	blocked "A Quarg ship broadcasts that they need a ship with at least <bunks> bunks for an emergency evacuation."
+
+	to offer
+		not "event: wanderers: kor mereti friendly"
+		has "First Contact: Karek Fornati: offered"
+		
+	on offer
+		conversation
+			`The blue flash of Quarg ships is nothing new as you travel Efreti space. Usually that is all you see of the enigmatic aliens. Today, however, you receive an urgent hail as you land.`
+			`	"Salutations Captain of the <ship>. Your most diligent efforts are required lest many families mourn the dead. An Efreti ship, the <EDC-ship>, was surveying <stopovers> when surprised by Mereti they were. Their ship crashed on the planet with many dead and they have asked for help to evacuate."`
+			choice
+				`	"I'll go there immediately."`
+					accept
+				`	"Can't the Quarg respond?"`
+					goto quarg
+				`	"I don't care about the Korath."`
+					goto careless
+			
+			label quarg
+			`	"We fought off the Mereti that attacked them but they had crashed before we could reach them. The gravity on that world requires us to have hardened environment suits to navigate wreckage while a Human would have few problems. You can reach them days before we could return from Kuwaru Efreti."`
```

Comment:
> The Quarg seem to have gone from "salutations," in the previous line to just speaking the same way as anyone else might?

I will rework this to match the style. With regards to changing Human to lowercase, every other species is identified as a proper noun by capitalization, why would we not do so when a Quarg is speaking?

Reply:
Could you provide some examples of this? I don't recall seeing human capitalized; I think it would be good to keep this consistent with the rest of the game.

Reply:
First contact with the Quarg in Efreti space: you think you recognize the word "Humani," and perhaps, "Eartha."
First contact with the Quarg they capitalize every species listed: We cannot step many steps on the worlds that the Hai, the Korath, the other species desire.